src/index.html

```diff
@@ -2,15 +2,50 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    data-qa="big">
+      <label>
+        <div class="search-bar">
+          <img
+          src="images/Search.svg"
+          alt="search-icon"
+          class="search-icon1">
+          <input
+          type="search"
+          class="search-big"
+          name="search-form"
+          placeholder="Try “Los Angeles”"
+          data-qa="keypress">
```

Comment:
```suggestion
          <img
            src="images/Search.svg"
            alt="search-icon"
            class="search-icon1"
          >
          <input
            type="search"
            class="search-big"
            name="search-form"
            placeholder="Try “Los Angeles”"
            data-qa="keypress"
          >
```

у атрибутво внутри тега +2 отступа

закрывающий тег переносится на след строку